src/main/scala/cromwell/util/Encryption.scala

```diff
@@ -0,0 +1,75 @@
+package cromwell.util
+
+import java.security.SecureRandom
+import javax.crypto.Cipher
+import javax.crypto.spec.{IvParameterSpec, SecretKeySpec}
+
+import org.apache.commons.codec.binary.Base64
+
+import scala.util.{Failure, Success, Try}
+
+case object Aes256Cbc {
+
+  val encryption = "AES"
+  val blockSize = 128
+  val keySize = 256
+  val paddingMode = "PKCS5Padding"
+  val cipherMode = "CBC"
+
+  val ranGen = new SecureRandom()
+
+  final def init(mode: Int, secretKey: Array[Byte], iv: Array[Byte]) = {
```

Comment:
I'm inclined to leave it as is, but there was a sweeping wave of voetization at work last week. Although it might be good to note that's what iv is in a couple of prominent places just to make it clear.